src/main/java/com/cloudbees/jenkins/support/impl/HeapUsageHistogram.java

```diff
@@ -84,23 +83,16 @@ private String getLiveHistogram() throws IOException {
     }
 
     private String getRawLiveHistogram() {
-        if (DISABLED) {
-            return new StringBuilder().append('\n')
-                    .append("Histogram generation is disabled. If you want to enable it, do either:")
-                    .append('\n')
-                    .append("* Add the system property: -Dcom.cloudbees.jenkins.support.impl.HeapUsageHistogram.DISABLED=false")
-                    .append('\n')
-                    .append("* Run from Script Console the line: com.cloudbees.jenkins.support.impl.HeapUsageHistogram.DISABLED=false")
-                    .toString();
-        }
         String result;
         try {
             ObjectName objName = new ObjectName("com.sun.management:type=DiagnosticCommand");
             MBeanServer platform = ManagementFactory.getPlatformMBeanServer();
             if (platform == null) {
                 return "N/A";
             }
-            result = (String) platform.invoke(objName, "gcClassHistogram", new Object[] {null}, new String[]{String[].class.getName()});
+            String[] params = FULLGC ? null : new String[] {"-all"};
```

Comment:
I don't see us getting much value from the histogram if it's capturing dead objects too.  Better to be live-only or off.